security-force-monitor/sfm-cms
# UI update on edit buttons

## Overview

This PR correctly highlights which editing page the user is on when they are editing an organization. E.g. if they are editing personnel, only the "Personnel" editing button will be active.

Connects #446 

### Demo

<img width="647" alt="Screen Shot 2019-08-15 at 4 50 58 PM" src="https://user-images.githubusercontent.com/6961258/63129348-e88e4a80-bf7c-11e9-9c4a-e5885e4e347b.png">

## Testing Instructions

- Choose an organization to edit.
- Navigate to the editing page.
- Click on each section button, confirming that only the section you're on is being highlighted.
- Test the behavior is consistent when both editing current entities and adding new ones.

- Click to create an organization. 
- Confirm the page loads and that only the "Basics" button is shown.
